componets/input/input.html

```diff
@@ -0,0 +1,410 @@
+              style="min-height: 299px"
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+        <div class="component_types" id="textArea">
+          <h2 class="component__name">Text Area</h2>
+          <p class="component__description">
+            Use class input__textarea to use textarea
+          </p>
+          <div class="components__example">
+            <div class="input__textarea">
+              <label for="review">Your Review of This site </label>
+              <textarea id="review"></textarea>
+            </div>
+          </div>
+          <div class="component_code">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=material&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=14px&lh=144%25&si=false&es=2x&wm=false&code=%2520%2509%2509%2520%2520%2520%2520%253Cdiv%2520class%253D%2522input__textarea%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522review%2522%253EYour%2520Review%2520of%2520This%2520site%2520%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ctextarea%2520id%253D%2522review%2522%253E%253C%252Ftextarea%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
+              style="min-height: 138px"
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+        <div class="component_types" id="sampleForm">
+          <h2 class="component__name">Sample Form</h2>
+          <p class="component__description">
+            Sample Registration Form Use class "form" to use this form
+          </p>
+          <div class="components__example">
+            <form action="./input.html" class="form">
+              <h2>Registration</h2>
+              <div class="input__box">
+                <label for="Email">Email</label>
+                <input id="Email" class="input" />
+              </div>
+              <div class="input__box">
+                <label for="Username">Username </label>
+                <input id="Username" class="input" />
+              </div>
+              <div class="input__box">
+                <label for="Password">Password</label>
+                <input id="Password" class="input" />
+              </div>
+              <div class="input__box">
+                <label for="ConfirmPassword">Confirm Password </label>
+                <input id="ConfirmPassword" class="input" />
+              </div>
+              <input type="submit" value="Submit" class="btn btn-primary" />
+            </form>
```

Comment:
In form inputs add "required" keyword then the form will not submit on empty inputs.

Reply:
yes i have done the changes